### ISSUE TITLE
Use CMake export

### ISSUE THREAD

**Issue by jlblancoc** [MRPT/mrpt#602]
Think whether it's worth changing the current `MRPTConfig.cmake` to use CMake standard "EXPORT" command. 
* Pros: should make it simpler to work with static vs dynamic libs (without all the ad-hoc CMake code currently present).
* Cons: well... the current solution "just works", despite being less elegant...

Inspirational example [here](https://github.com/RainerKuemmerle/g2o/compare/Export_cmake_Configuration#diff-dfdf08900eb0292ea317b9f75ef332ed).

**Comment by jolting** [comment]
I think using the cmake convention will prevent problems in the future and avoid some potential bit rot.

**Comment by jlblancoc** [comment]
Another source of inspiration for a minimal working case [here](https://github.com/strasdat/Sophus/pull/120/files).

**Comment by jlblancoc** [comment]
WIP in branch `cmake-config-targets`.

**Comment by jlblancoc** [comment]
Reopening, since Debian pkgs are now broken:

```
 Install configuration: "Release"
CMake Error at cmake_install.cmake:49 (file):
  file INSTALL cannot find
  "/<<PKGBUILDDIR>>/obj-x86_64-linux-gnu/unix-install/MRPTConfig.cmake".


make[2]: *** [Makefile:65: install] Error 1
```

**Comment by jlblancoc** [comment]
Fixed by cbc00a7f1058a5874bead8ec42a2a22338a2a20f
